lib/Ohmbrewer_Screen.cpp

```diff
@@ -15,6 +16,14 @@ Ohmbrewer::Screen::Screen(uint8_t CS,
                           uint8_t RST,
                           std::deque< Ohmbrewer::Equipment* >* sprouts) : Adafruit_ILI9341(CS, RS, RST) {
     _sprouts = sprouts;
+	/** 
+    *  For better pressure precision, we need to know the resistance
+	*  between X+ and X- Use any multimeter to read it
+	*  Using value of 300 ohms across the X plate
```

Comment:
Actually, this is using 285 ohms... Either fix the comment or fix the setting on line 24 :wink: